src/overlays/actors/ovl_Bg_Jya_Megami/z_bg_jya_megami.h

```diff
@@ -4,11 +4,28 @@
 #include <ultra64.h>
 #include <global.h>
 
+#define BGJYAMEGAMI_NUMPIECES 13
+
 struct BgJyaMegami;
 
+typedef void (*BgJyaMegamiActionFunc)(struct BgJyaMegami*, GlobalContext*);
+
+typedef struct BgJyaMegami_Piece {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ Vec3f vel;
+    /* 0x18 */ s16 rotVelX;
+    /* 0x1A */ s16 rotVelY;
+} BgJyaMegami_Piece; // size = 0x1C
```

Comment:
Type names usually don't include any underscores.
```suggestion
} BgJyaMegamiPiece; // size = 0x1C
```